### PR TITLE
use semver as annotation version

### LINKED ISSUE #468: Use semver version in the resource annotation to manage upgrade

### PR DESCRIPTION
**Pull request by horis233**
fix #468

### CODE REVIEW

**Comment by Daniel-Fan** [comment]
@horis233 

Could you update the `annotation.version` for the CRs of cert-manager as well? Thank you!
https://github.com/IBM/ibm-common-service-operator/blob/master/controllers/certmanager/cert_cr.go#L25
https://github.com/IBM/ibm-common-service-operator/blob/master/controllers/certmanager/cert_cr.go#L43
https://github.com/IBM/ibm-common-service-operator/blob/8a3fcadce0019cec80da238091687b9256e42ac5/controllers/certmanager/cert_cr.go#L60

File: controllers/certmanager/cert_cr.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ apiVersion: certmanager.k8s.io/v1alpha1
 kind: Certificate
 metadata:
   annotations:
-    version: "1"
+    version: "3.8.0"
```
Comment (horis233) [inline controllers/certmanager/cert_cr.go:60]: @Daniel-Fan They are updated

Comment (Daniel-Fan) [reply controllers/certmanager/cert_cr.go:60]: Thank so you much!

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**